<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageSetDialog</class>
 <widget class="QDialog" name="LanguageSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonLagSetEnSel">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>145</y>
     <width>450</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonLagSetSimpChSel">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>255</y>
     <width>450</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>简体中文</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonLagSetSimpCh">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>255</y>
     <width>450</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>简体中文</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonLagSetEn">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>145</y>
     <width>450</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonLagSetTraCh">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>365</y>
     <width>450</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>繁體中文</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonLagSetTraChSel">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>365</y>
     <width>450</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>繁體中文</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonLagSetReturn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>470</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonLagSetEn</tabstop>
  <tabstop>pushButtonLagSetEnSel</tabstop>
  <tabstop>pushButtonLagSetSimpCh</tabstop>
  <tabstop>pushButtonLagSetSimpChSel</tabstop>
  <tabstop>pushButtonLagSetTraCh</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
